shared/actions/push.js

```diff
@@ -52,10 +52,12 @@ function* pushNotificationSaga(notification: PushGen.NotificationPayload): Saga.
     if (payload.type === 'chat.newmessageSilent_2') {
       logger.info('Push notification: silent notification received')
       try {
+        // $ForceType
```

Comment:
What is `$ForceType`? I can't seem to find docs for it anywhere

Reply:
we made it to semantically capture that we're doing a 'cast'

Reply:
https://github.com/keybase/client/blob/master/shared/.flowconfig#L32